src/cpu.h

```diff
@@ -66,20 +66,22 @@ extern CPU cpu;
 #define CPU_REG_F   cpu.AF.words.lo
 
 void cpu_init(void);
-__attribute__((pure)) uint16_t cpu_get_cycle_count(void);
 void cpu_step(void);
 
-__attribute__((pure)) uint16_t cpu_get_PC(void);
 void cpu_print_registers(void);
 
-__attribute((always_inline)) inline uint8_t get_flag_bit(Flag f) {
+__attribute((always_inline)) inline uint8_t get_flag(Flag f) {
     return (CPU_REG_F >> f) & 1;
 }
 
 __attribute((always_inline)) inline void set_flag(uint8_t bit, Flag f) {
     CPU_REG_F |= bit << f;
 }
 
+__attribute((always_inline)) inline void clear_flag(Flag f) {
+    CPU_REG_F &= (uint8_t) ~(1 << f);
```

Comment:
```suggestion
    set_flag(1,f);
```
This might be clearer as to what's going on instead of doing bitshifts again